### DISCUSSION TITLE
Add the address alias in the order page

### DISCUSSION THREAD

**Discussion by AudaceV** [PrestaShop/PrestaShop#34765] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Missing the tittle address in order page.
![6GITgzR](https://github.com/PrestaShop/PrestaShop/assets/152642054/ce817017-9582-464e-881c-aae44613d14c)
Should be appears the title like this
![pUZjkjP](https://github.com/PrestaShop/PrestaShop/assets/152642054/6707ce57-db80-4f30-82f9-d08ea9fd8cb3)

### Expected behavior

Display on the list  %adress alias% - %adress%
 
### Steps to reproduce

Sell>order>order

1. Selected another address

### PrestaShop version(s) where the bug happened

8.0 and 8.1

**Comment by florine2623** [comment]
Hello @AudaceV ,

This is how it used to work in PS 1.7.8 also. This is the expected behavior.

**Comment by florine2623** [comment]
ping @PrestaShop/product-council ,

What do you think about implementing this new feature ?

**Comment by AudaceV** [comment]
Hi @florine2623,
I think that was better in 1.6. and this feature should be update in PS 8

**Comment by MatShir** [comment]
You would like to see the alias and the full adress ?

**Reply by MatShir** [reply]
I transferred the issue into discussion. It is more convenient to exchange improvement ideas on it

**Reply by AudaceV** [reply]
Hi, @MatShir,
I would like to see the alias like in PS 1.6

**Reply by MatShir** [reply]
I don't remember. I will have a look

**Reply by MatShir** [reply]
It was %adress alias% - %adress%. thanks for the feedback, it does make sense

**Reply by AudaceV** [reply]
Please, see here:
![A9a6HAh](https://github.com/PrestaShop/PrestaShop/assets/152642054/2721dac8-da1c-4ae4-b824-615e3426f4dc)

**Reply by MatShir** [reply]
Thanks, other feedback on the new order page you would like to share ?

**Reply by AudaceV** [reply]
Sure,
I finded this issue also https://github.com/PrestaShop/PrestaShop/issues/34751
If you check this issue, remember, the state of order must be "PAYMENT ACCEPTED"

**Comment by kpodemski** [comment]
I submitted a PR to add an alias: https://github.com/PrestaShop/PrestaShop/pull/34833
